src/app/features/auth/guards/authenticated.guard.ts

```diff
@@ -0,0 +1,7 @@
+import { inject } from '@angular/core';
+import { CanMatchFn, Router } from '@angular/router';
+import { AuthService } from '@features/auth/services/auth/auth.service';
+
+export const authenticatedGuard: CanMatchFn = () => {
+  return !inject(AuthService).isLoggedIn() || inject(Router).createUrlTree(['/home']);
+};
```

Comment:
I suggest instead of two autoGuard and authenticatedGuard to make one like this

```
export const authGuard: CanMatchFn = (route: Route) => {
  const authService = inject(AuthService);
  const router = inject(Router);

  const authPath = ['registration', 'login'];

  if (authPath.includes(route.path ?? '')) {
    return !authService.isLoggedIn() || router.createUrlTree(['/home']);
  }
  return authService.isLoggedIn() || router.createUrlTree(['/login']);
};
```